both/_round.js

```diff
@@ -0,0 +1,175 @@
+
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+Round = function(doc) {
+  _.extend(this, doc);
+};
+_.extend(Round.prototype, {
+
+  format: function() {
+    return wca.formatByCode[this.formatCode];
+  },
+  properties: function() {
+    return wca.roundByCode[this.roundCode];
+  },
+  eventName: function() {
+    return this.eventCode ? wca.eventByCode[this.eventCode].name : '';
+  },
+  eventSolveTimeFields: function() {
```

Comment:
Maybe. This API seemed nicer to me. There are only two fields, and they seem unrelated, while the others look more like coherent structures.

But that just me winging it. I have neither a strong opinion or a deep understanding.